source/speakers/index.html.erb

```diff
@@ -9,6 +9,39 @@ created_by:
   name: Mike Rogers
 ---
 
+<div class="speaker__list">
+
+  <% all_speakers.each do |speaker| %>
+    <div class="speaker">
+      <div class="speaker__avatar">
+        <%= image_tag speaker.avatar_url, alt: "#{speaker.name} avatar" %>
+      </div>
+
+      <div class="speaker__details">
+        <h2 class="speaker__name"><%= speaker.name %></h2>
+        <div class="speaker__socials">
+          <% link_to "https://twitter.com/#{speaker.twitter_username}" do %>
+            <%= image_tag 'twitter.svg', width: 24, height: 24 %>
+          <% end %>
+
+          <% link_to "https://github.com/#{speaker.github_username}" do %>
+            <%= image_tag 'github.svg', width: 24, height: 24 %>
+          <% end %>
+        </div>
+
+        <p><%= speaker.bio %></p>
+
+        <p>Talks: <%= speaker.talks.count %></p>
+
+        <p>
+          <%= link_to speaker.blog, speaker.blog %>
+        </p>
+      </div>
+    </div>
+  <% end %>
+
+</div>
+
 <% data.coverage.sort_by { |key, value| -key.to_i }.each do |year, month_and_talks| %>
```

Comment:
I'm guessing this list of talks was a placeholder just to get something up.  I think a version of this list per speaker (some have definitely spoken several times over the years) would make sense, but I don't think a full version is that useful.

Reply:
Hah yes - I was experimenting with how the coverage data is stored & left it in 😅